index.js

```diff
@@ -1,2 +1,876 @@
+  "📴",
+  "📵",
+  "📶",
+  "📷",
+  "📹",
+  "📺",
+  "📻",
+  "📼",
+  "🔀",
+  "🔁",
+  "🔂",
+  "🔃",
+  "🔄",
+  "🔅",
+  "🔆",
+  "🔇",
+  "🔈",
+  "🔉",
+  "🔊",
+  "🔋",
+  "🔌",
+  "🔍",
+  "🔎",
+  "🔏",
+  "🔐",
+  "🔑",
+  "🔒",
+  "🔓",
+  "🔔",
+  "🔕",
+  "🔖",
+  "🔗",
+  "🔘",
+  "🔙",
+  "🔚",
+  "🔛",
+  "🔜",
+  "🔝",
+  "🔞",
+  "🔟",
+  "🔠",
+  "🔡",
+  "🔢",
+  "🔣",
+  "🔤",
+  "🔥",
+  "🔦",
+  "🔧",
+  "🔨",
+  "🔩",
+  "🔪",
+  "🔫",
+  "🔬",
+  "🔭",
+  "🔮",
+  "🔯",
+  "🔰",
+  "🔱",
+  "🔲",
+  "🔳",
+  "🔴",
+  "🔵",
+  "🔶",
+  "🔷",
+  "🔸",
+  "🔹",
+  "🔺",
+  "🔻",
+  "🔼",
+  "🔽",
+  "🕐",
+  "🕑",
+  "🕒",
+  "🕓",
+  "🕔",
+  "🕕",
+  "🕖",
+  "🕗",
+  "🕘",
+  "🕙",
+  "🕚",
+  "🕛",
+  "🕜",
+  "🕝",
+  "🕞",
+  "🕟",
+  "🕠",
+  "🕡",
+  "🕢",
+  "🕣",
+  "🕤",
+  "🕥",
+  "🕦",
+  "🕧",
+  "🗻",
+  "🗼",
+  "🗽",
+  "🗾",
+  "🗿",
+  "😀",
+  "😁",
+  "😂",
+  "😃",
+  "😄",
+  "😅",
+  "😆",
+  "😇",
+  "😈",
+  "😉",
+  "😊",
+  "😋",
+  "😌",
+  "😍",
+  "😎",
+  "😏",
+  "😐",
+  "😑",
+  "😒",
+  "😓",
+  "😔",
+  "😕",
+  "😖",
+  "😗",
+  "😘",
+  "😙",
+  "😚",
+  "😛",
+  "😜",
+  "😝",
+  "😞",
+  "😟",
+  "😠",
+  "😡",
+  "😢",
+  "😣",
+  "😤",
+  "😥",
+  "😦",
+  "😧",
+  "😨",
+  "😩",
+  "😪",
+  "😫",
+  "😬",
+  "😭",
+  "😮",
+  "😯",
+  "😰",
+  "😱",
+  "😲",
+  "😳",
+  "😴",
+  "😵",
+  "😶",
+  "😷",
+  "😸",
+  "😹",
+  "😺",
+  "😻",
+  "😼",
+  "😽",
+  "😾",
+  "😿",
+  "🙀",
+  "🙁",
+  "🙂",
+  "🙅",
+  "🙆",
+  "🙇",
+  "🙈",
+  "🙉",
+  "🙊",
+  "🙋",
+  "🙌",
+  "🙍",
+  "🙎",
+  "🙏",
+  "🚀",
+  "🚁",
+  "🚂",
+  "🚃",
+  "🚄",
+  "🚅",
+  "🚆",
+  "🚇",
+  "🚈",
+  "🚉",
+  "🚊",
+  "🚋",
+  "🚌",
+  "🚍",
+  "🚎",
+  "🚏",
+  "🚐",
+  "🚑",
+  "🚒",
+  "🚓",
+  "🚔",
+  "🚕",
+  "🚖",
+  "🚗",
+  "🚘",
+  "🚙",
+  "🚚",
+  "🚛",
+  "🚜",
+  "🚝",
+  "🚞",
+  "🚟",
+  "🚠",
+  "🚡",
+  "🚢",
+  "🚣",
+  "🚤",
+  "🚥",
+  "🚦",
+  "🚧",
+  "🚨",
+  "🚩",
+  "🚪",
+  "🚫",
+  "🚬",
+  "🚭",
+  "🚮",
+  "🚯",
+  "🚰",
+  "🚱",
+  "🚲",
+  "🚳",
+  "🚴",
+  "🚵",
+  "🚶",
+  "🚷",
+  "🚸",
+  "🚹",
+  "🚺",
+  "🚻",
+  "🚼",
+  "🚽",
+  "🚾",
+  "🚿",
+  "🛀",
+  "🛁",
+  "🛂",
+  "🛃",
+  "🛄",
+  "🛅",
+  "‼",
+  "⁉",
+  "™",
+  "ℹ",
+  "↔",
+  "↕",
+  "↖",
+  "↗",
+  "↘",
+  "↙",
+  "↩",
+  "↪",
+  "#⃣",
+  "⌚",
+  "⌛",
+  "⏩",
+  "⏪",
+  "⏫",
+  "⏬",
+  "⏰",
+  "⏳",
+  "Ⓜ",
+  "▪",
+  "▫",
+  "▶",
+  "◀",
+  "◻",
+  "◼",
+  "◽",
+  "◾",
+  "☀",
+  "☁",
+  "☎",
+  "☑",
+  "☔",
+  "☕",
+  "☝",
+  "☺",
+  "♈",
+  "♉",
+  "♊",
+  "♋",
+  "♌",
+  "♍",
+  "♎",
+  "♏",
+  "♐",
+  "♑",
+  "♒",
+  "♓",
+  "♠",
+  "♣",
+  "♥",
+  "♦",
+  "♨",
+  "♻",
+  "♿",
+  "⚓",
+  "⚠",
+  "⚡",
+  "⚪",
+  "⚫",
+  "⚽",
+  "⚾",
+  "⛄",
+  "⛅",
+  "⛎",
+  "⛔",
+  "⛪",
+  "⛲",
+  "⛳",
+  "⛵",
+  "⛺",
+  "⛽",
+  "✂",
+  "✅",
+  "✈",
+  "✉",
+  "✊",
+  "✋",
+  "✌",
+  "✏",
+  "✒",
+  "✔",
+  "✖",
+  "✨",
+  "✳",
+  "✴",
+  "❄",
+  "❇",
+  "❌",
+  "❎",
+  "❓",
+  "❔",
+  "❕",
+  "❗",
+  "❤",
+  "➕",
+  "➖",
+  "➗",
+  "➡",
+  "➰",
+  "➿",
+  "⤴",
+  "⤵",
+  "⬅",
+  "⬆",
+  "⬇",
+  "⬛",
+  "⬜",
+  "⭐",
+  "⭕",
+  "0⃣",
+  "〰",
+  "〽",
+  "1⃣",
+  "2⃣",
+  "㊗",
+  "㊙",
+  "3⃣",
+  "4⃣",
+  "5⃣",
+  "6⃣",
+  "7⃣",
+  "8⃣",
+  "9⃣",
```

Comment:
You're right!

It could be possible that alt attribute content the emoji? If this is true you don't need trim the `li` tag, just get `alt` value.